### PR TITLE
Remove deprecated as_pairs function.

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

File: iris_grib/__init__.py [hunk 1]
```diff
@@ -1028,19 +1028,6 @@ def save_grib2(cube, target, append=False, **kwargs):
     save_messages(messages, target, append=append)
 
 
-def as_pairs(cube):
-    """
-    .. deprecated:: 1.10
```
Comment (dkillick) [inline iris_grib/__init__.py]: Ooh, this is a little more tricky: Iris v1.10 is the current version so can we say this has observed its deprecation period?

Answers on a postcard :postbox:

**Comment by dkillick** [comment]
Again, relates to #8, I'm guessing?

**Comment by dkillick** [comment]
@bjlittle you've got merge conflicts now that I've merged #29...

File: iris_grib/__init__.py (hunk 1 above)
Comment (bjlittle) [inline iris_grib/__init__.py]: My take on it is that it's deprecated in `iris` as a fall-back implementation. `iris-grib` on the otherhand doesn't need to honour that deprecation. It's a brand new world, so purge the deprecations as per #8

Comment (dkillick) [inline iris_grib/__init__.py]: Works for me!

**Comment by bjlittle** [comment]
@dkillick All rebased and lookin' good ... :sunglasses:

**Comment by dkillick** [comment]
Excellent! More deprecations squashed :bug: Thanks @bjlittle

**Pull request merged**